### ISSUE TITLE
Old albums randomly set as monitored

### ISSUE THREAD

**Issue by wizzy99** [Lidarr/Lidarr#354]
Version 0.3.0.403
Mono Version 5.12.0.226
DB Migration 16

I imported my library a couple of weeks ago, and chose "future only" to monitor.  There were a few artists where that wasn't respected, but overall it was.

I went into my wanted folder this weekend and found that the list had exploded to > 6000 albums, most of them in the past.  No additional importing had been done, so I'm unsure why all these old albums were flipped to monitored status. 

[AB#479](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/479)

**Comment by wizzy99** [comment]
Still not sure what happened, but I was able to clean things up by launching sqlite and running

update albums
set monitored = 0
where monitored = 1 and releasedate < Datetime('2018-01-01 00:00:00');

That unmonitored everything that was released pre-12/31/17.

**Comment by Qstick** [comment]
Is there a scenario where you can constantly reproduce this? If so what?

**Comment by wizzy99** [comment]
It seems to be either when a new "old" album appears on Musicbrainz, or the metadata profile is updated to include additional content types and then the artist is refreshed.

Mentally I've equated "future" with not-yet-released items, but it seems that anything new that is added to an artist is being flagged as wanted.

**Comment by sasjafor** [comment]
This happens to me for many many artists. I always chose 'Existing Albums' when importing, but anytime that a release is added on Musicbrainz, it gets monitored by Lidarr. This is unwanted and unexpected behaviour, as it downloads or tries to download albums that I don't want.

I see the description of 'Existing Albums' is "Existing Albums
Monitor albums that have files or have not released yet". 
However, not all albums that are added to Musicbrainz are newly released albums.

From a naming perspective, I would have excpected 'Existing Albums' to only mean what I already have, without anything getting added without me doing it. However, I think what I would have wanted for my use case is 'None'. Where I then only add albums to be tracked and downloaded manually.

**Comment by sasjafor** [comment]
Also, if there is a way that I could change the monitoring from existing to something that doesn't monitor "new" albums, I would like to know about it.
I know that I could just unmonitor all artists, but I want to keep everything that is monitored now, so that I can only uncheck what I didn't add myself.

**Comment by Qstick** [comment]
Thanks for the additional context @sasjafor, I’ll look at getting this fixed with #580 as it contains some changes to this logic

**Comment by Redundant1** [comment]
@Qstick Did anything come of what @sasjafor mentioned?  It's referenced in #580, but as this is still open, it would appear it didn't get fixed.  I saw that @wieluk brought the same issue up as a new bug under #1831 and I commented there, as well, but it doesn't appear that anyone has noticed.  I'll wait to see if anyone takes notice here.  If not, I'll post it as a new issue in a week or so.  Here's what I wrote:

This is an issue for me, as well, but for a different reason. I only put albums I own in Lidarr and I want it to grab those albums and upgrade them until the cutoff is met. So, I may have only 2 albums from a given artist being monitored. If a new album is released for that artist, that new one also gets monitored, which I don't want. If I unmonitor the artist, then the albums I have monitored will no longer be upgraded, even if the cutoff is unmet.
